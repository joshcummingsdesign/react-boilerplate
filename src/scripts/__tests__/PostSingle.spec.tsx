import * as React from 'react';
import { shallow } from 'enzyme';
import PostSingle from '../components/PostSingle';
const posts = require('../__mocks__/posts.json');
const images = require('../__mocks__/images.json');

test('PostSingle renders correctly', () => {
  const component = shallow(<PostSingle post={posts[0]} image={images[0]} />);
  expect(component).toMatchSnapshot();
});
