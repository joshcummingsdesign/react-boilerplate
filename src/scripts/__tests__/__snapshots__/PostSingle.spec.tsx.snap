// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostSingle renders correctly 1`] = `
<div
  className="post-single"
>
  <h1>
    sunt aut facere repellat provident occaecati excepturi optio reprehenderit
  </h1>
  <p>
    quia et suscipit
suscipit recusandae consequuntur expedita et cum
reprehenderit molestiae ut ut quas totam
nostrum rerum est autem sunt rem eveniet architecto
  </p>
  <img
    alt="accusamus beatae ad facilis cum similique qui sunt"
    src="https://via.placeholder.com/150/92c952"
  />
</div>
`;
